apps/documentation/src/routes/__root.tsx

```diff
@@ -73,18 +73,27 @@ export default function RootRoute({ children }: RootRouteProps): JSX.Element {
           sizes="180x180"
           href="/apple-touch-icon.png"
         />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="32x32"
-          href="/favicon-32x32.png"
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="16x16"
-          href="/favicon-16x16.png"
-        />
+        {pathname.startsWith('/documentation') ? (
```

Comment:
Could you please apply this change to all the pages and remove the not used any more files from the `public/` folder?

Reply:
@NayanVR you should remove this condition and apply the new favicon to all the pages

Reply:
Okay got it. Should I apply new favicon to hero page (landing page) also?

Reply:
Yes

Reply:
Updated the commit